src/Phobos.Kafka/src/Petabridge.Phobos.Kafka.Producer/AkkaService.cs

```diff
@@ -113,6 +153,60 @@ public AkkaActors(ActorSystem sys)
         public IActorRef RouterForwarderActor { get; }
         
         public IActorRef ProducerActor { get; }
+
+        private Source<IResults<TKey, TValue, NotUsed>, IActorRef> ProducerActorRefWithCommit<TKey, TValue>(
```

Comment:
Not used in this sample, an example code of a generic way for creating an actor ref source.